taxonomy/admin.py

```diff
@@ -138,3 +138,14 @@ class RefreshProgramSkillsConfigAdmin(admin.ModelAdmin):
     """
     RefreshProgramSkillsConfig admin view.
     """
+
+
+@admin.register(Industry)
+class IndustryAdmin(admin.ModelAdmin):
+    """
+    Industry admin view.
+    """
+
+    list_display = ('id', 'code', 'name',)
+    search_fields = ('name', )
```

Comment:
Should not we enable search based on code as well?

Reply:
Codes are simple numbers, I do not think it makes sense to enable search based on them. People will not need to search based on code.